src/components/app.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import BrowserRouter from 'react-router-dom/BrowserRouter';
+import { renderRoutes } from 'react-router-config';
+
+import routes from './routes';
+
+const App = () => {
+    return (
+        <BrowserRouter>
+            {renderRoutes(routes)}
+        </BrowserRouter>
+    )
+}
+
+render(<App/>, document.querySelector('#root'));
```

Comment:
You might want to look into ReactDom's `hydrate()` method. https://reactjs.org/docs/react-dom.html#hydrate